### ISSUE TITLE
Arabic Mirza font has some issues when typed in Word

### ISSUE THREAD

**Issue by Briox** [google/fonts#1828]
Hello,

Thank you for this great font. However, sometimes I use it to write arabic text in Microsoft Word and it has major problem that the lower part of the text is cropped and the dots of the letters with lower dots can not be shown. I have tried several spacing values but it doesn't seem to solve the problem. See the following screenshot comparing the three types of Mirza font in the upper part from the horizontal line  and Arial and other font in the lower part all with default spacing. 
![capture3](https://user-images.githubusercontent.com/47121050/51861273-da643e00-234c-11e9-99c7-d4102177ec62.PNG)

Also, see how this affects the reading of the text makes it unreadable, keep in mind that if the paragraph spacing is increased only white space added nothing happen to the cropping, I have circled some of the differences of a regular font like Arial (lower) and Mirza (upper) in the same conditions.
![capture8](https://user-images.githubusercontent.com/47121050/51862800-7774a600-2350-11e9-9cc8-21796b2e09f0.PNG)

I really liked this font, please consider solving the balance and cropping problem. In addition, the font works fine in web pages as illustrated in the test web page http://tarobish.github.io/Mirza/html/live-testing.html?_sm_byp=iVVBRLPms4LSPJms however it has abnormal selection shadow see the following screenshot:
![capture6](https://user-images.githubusercontent.com/47121050/51861863-57dc7e00-234e-11e9-864f-57c38d805fd6.PNG)

It is also noticed that the English text using Mirza font has nicer selection shadow and no cropping. See:
![capture7](https://user-images.githubusercontent.com/47121050/51862082-ddf8c480-234e-11e9-847e-0d8c6799fbc9.PNG)

Your help is appreciated, thanks,

**Comment by laerm0** [comment]
Hello @Briox. I have updated the vertical metrics of this file to match our spec more accurately and there are improvements. How does this look to you? I used the testing page you used (thanks for supplying that).

![screen region 2019-02-05 at 15 04 12](https://user-images.githubusercontent.com/10780161/52301051-7b5b8600-2957-11e9-8dd5-0667636ab086.png)

![screen region 2019-02-05 at 15 04 20](https://user-images.githubusercontent.com/10780161/52301056-7eef0d00-2957-11e9-8094-194045789fda.png)
